cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
```

Comment:
Nice work! However, I would like to suggest you adhere to the MVVM pattern your code is trying to achieve by separating objects into three distinct groups.  

Refer to the other comments for some guides.